noxfile.py

```diff
@@ -79,22 +81,44 @@ def update_constraints(session: nox.Session) -> None:
         # CUSTOM_COMPILE_COMMAND is a pip-compile option that tells users how to
         # regenerate the constraints files
         env["UV_CUSTOM_COMPILE_COMMAND"] = f"nox -s {session.name}"
+        output_file = resources / f"constraints-python{python_version.replace('.', '')}.txt"
         session.run(
             "uv",
             "pip",
             "compile",
             f"--python-version={python_version}",
             "--upgrade",
-            "cibuildwheel/resources/constraints.in",
-            f"--output-file=cibuildwheel/resources/constraints-python{python_version.replace('.', '')}.txt",
+            resources / "constraints.in",
+            f"--output-file={output_file}",
             env=env,
         )
-    RESOURCES = DIR / "cibuildwheel" / "resources"
+
     shutil.copyfile(
-        RESOURCES / "constraints-python312.txt",
-        RESOURCES / "constraints.txt",
+        resources / "constraints-python312.txt",
+        resources / "constraints.txt",
     )
 
+    build_platforms = nox.project.load_toml(resources / "build-platforms.toml")
+    pyodides = build_platforms["pyodide"]["python_configurations"]
+    for pyodide in pyodides:
+        python_version = ".".join(pyodide["version"].split(".")[:2])
+        pyodide_version = pyodide["pyodide_version"]
+        output_file = resources / f"constraints-pyodide{python_version.replace('.', '')}.txt"
+        tmp_file = Path(session.create_tmp()) / "constraints-pyodide.in"
+        tmp_file.write_text(
+            f"auditwheel-emscripten\nbuild[virtualenv]\npyodide-build=={pyodide_version}"
```

Comment:
```suggestion
            f"auditwheel-emscripten\nbuild[virtualenv]\npip\npyodide-build=={pyodide_version}"
```

This would require re-running `nox -s update_constraints`

Reply:
I'll swap auditwheel-emscripten for pip, then. That's currently required by and only used through pyodide build, so listing it again here isn't needed.